cypress-custom/integration/swapMod.ts

```diff
@@ -40,15 +40,20 @@ describe('Swap (mod)', () => {
   })
 
   it('zero output amount', () => {
-    cy.get('#swap-currency-output .token-amount-input').clear().type('0.0', { delay: 400 }).should('have.value', '0.0')
+    cy.get('#swap-currency-output .token-amount-input')
+      // When `.clear() doesn't work, brute force it with the input below.
+      // From https://stackoverflow.com/a/65918033/1272513
+      .type('{selectall}{backspace}{selectall}{backspace}')
+      .type('0.0', { delay: 400 })
+      .should('have.value', '0.0')
   })
 
   it('can swap Native for DAI', () => {
     cy.get('#swap-currency-output .open-currency-select-button').click()
     cy.get('.token-item-0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735').should('be.visible')
     cy.get('.token-item-0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735').click({ force: true })
     cy.get('#swap-currency-input .token-amount-input').should('be.visible')
-    cy.get('#swap-currency-input .token-amount-input').type('0.001', { delay: 400, force: true })
+    cy.get('#swap-currency-input .token-amount-input').clear().type('0.5', { delay: 400, force: true })
```

Comment:
So we not need the brute force clear here as well?

Reply:
For some reason, not everywhere is needed.
Did not try hard enough to see whether that would break without it